src/OrbitGl/CaptureViewElementTester.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "CaptureViewElementTester.h"
+
+#include <gtest/gtest.h>
+
+void orbit_gl::CaptureViewElementTester::RunTests(CaptureViewElement* element) {
+  TestWidthPropagationToChildren(element);
+}
+
+void orbit_gl::CaptureViewElementTester::TestWidthPropagationToChildren(
+    CaptureViewElement* element) {
+  element->SetWidth(100);
```

Comment:
Nit: Just personal preference. I would like to use `kWidth` and `kUpdatedWidth` instead of 100 and 50.